From 16fa31708d6b6af047352d1a78a7eab823b63221 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Mon, 9 Apr 2018 17:06:37 -0400
Subject: [PATCH] Add font as valid media type

This was codified in RFC 8081, and some desktop apps are using
this now.
---
 src/mimeutils.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mimeutils.c b/src/mimeutils.c
index 0833fdb..bddc7a0 100644
--- a/src/mimeutils.c
+++ b/src/mimeutils.c
@@ -52,9 +52,9 @@ static const char *known_old_fdo_media_types[] = {
   "x-directory"
 };
 
-/* Defined in RFC 2045/2046 and RFC 2077 */
+/* Defined in RFC 2045/2046, RFC 2077 and RFC 8081 */
 static const char *registered_discrete_media_types[] = {
-  "application", "audio", "image", "model", "text", "video"
+  "application", "audio", "font", "image", "model", "text", "video"
 };
 
 /* Defined in RFC 2045/2046 */
-- 
2.17.0

